js/Heap.js

```diff
@@ -15,29 +16,32 @@ import Package from 'react-native-package';
  * https://github.com/negativetwelve/react-native-package
  */
 export default Package.create({
-  json: require('../package.json'),
+  json: require("../package.json"),
   nativeModule: NativeModules.RNHeap,
   enabled: Platform.select({
     ios: true,
+    android: true
   }),
-  export: (Heap) => ({
+  export: Heap => ({
     // App Properties
-    setAppId: (appId) => Heap.setAppId(appId),
+    setAppId: appId => Heap.setAppId(appId),
 
     // User Properties
-    identify: (identity) => Heap.identify(identity),
-    addUserProperties: (properties) => Heap.addUserProperties(properties),
+    identify: identity => Heap.identify(identity),
+    addUserProperties: properties => Heap.addUserProperties(properties),
 
     // Event Properties
-    addEventProperties: (properties) => Heap.addEventProperties(properties),
-    removeEventProperty: (property) => Heap.removeEventProperty(property),
+    addEventProperties: properties => Heap.addEventProperties(properties),
+    removeEventProperty: property => Heap.removeEventProperty(property),
     clearEventProperties: () => Heap.clearEventProperties(),
 
     // Events
     track: (event, payload) => Heap.track(event, payload),
 
-    // Config
-    enableVisualizer: () => Heap.enableVisualizer(),
-    changeInterval: (interval) => Heap.changeInterval(interval),
-  }),
+    // Redux middleware
+    reduxMiddleware: store => next => action => {
```

Comment:
Are we including this in the v0?  If not, let's remove this (along with the `flat` dep).

Reply:
I'd rather leave it in and see if someone is willing to try it out and give us feedback.